.buildkite/snapshot.yml

```diff
@@ -10,7 +10,5 @@ steps:
       - "**/target/*"
 
 notify:
-  - slack:
-    if: build.state == "failed"
-      channels:
-        - "#apm-agent-java"
+  - slack: "#apm-agent-java"
+    if: 'build.state != "passed"'
```

Comment:
I used single quotes as I've found some examples in the elastic org that use them

<img width="770" alt="image" src="https://user-images.githubusercontent.com/2871786/219031841-1b501f9c-2bde-4074-9287-debf1c04d526.png">


Although I've found also some others that don't use them

<img width="628" alt="image" src="https://user-images.githubusercontent.com/2871786/219031988-f1213def-2fc6-4bc1-9842-eac142da1ee8.png">